### ISSUE TITLE
Does this mod still have compatibility with 'Player Revive'?

### ISSUE THREAD

**Issue by tacusin** [ichttt/FirstAid#44]
First Aid-1.6.3
Player Revive-1.2.23

Just tried the latest versions of both and upon being revived it seems to break first aid. Bandages and Plaster Seem to become nonfunctional and when you use them they cooldown before using them again seems to get stuck at 18 seconds. And all health segments are at 0 after being revived. And even though all health segments are at 0 it doesn't seem be apply any debuffs from it.

**Comment by ichttt** [comment]
Oh not again...

**Comment by tacusin** [comment]
Should I leave an issue report on Player Revive as well? Or will just here be fine?

**Comment by ichttt** [comment]
Here is fine, I'll test when I have some time

**Comment by ichttt** [comment]
Waiting on PlayerRevive PR to be pulled (https://github.com/CreativeMD/PlayerRevive/pull/26)

**Comment by tacusin** [comment]
Seems he is back and implemented the changes :D
[image](https://user-images.githubusercontent.com/5170345/55829030-53151280-5ad3-11e9-9672-8d5ffb5e2cc9.png)

**Comment by ichttt** [comment]
Should work now once again. You need to use lastest PlayerRevive AND FirstAid though.

**Comment by tacusin** [comment]
I had already installed the latest of PlayerRevive when I noticed it had been updated. Also after reviving the health debuff effects don't seem to take effect until after the body section that causes the debuff undergoes a health update with either a bandage or plaster. Is this a intended "adrenaline" effect after a near-death experience. Or an unintended "feature/bug". Either way only having half a heart for body/head will be quite the experience to survive anyways x3
